pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>me.raindropz.mc</groupId>
-    <artifactId>npcommand</artifactId>
+    <artifactId>NPCommand</artifactId>
```

Comment:
Changing artifactId after its already existed for a while is an odd choice. Capitalizing it is an even odder choice.

Reply:
The capitalization is to match the project name, which is a standard convention when building a project into an artifact

Reply:
https://maven.apache.org/guides/mini/guide-naming-conventions.html

> artifactId is the name of the jar without version. If you created it, then you can choose whatever name you **want with lowercase letters** and no strange symbols. If it's a third party jar, you have to take the name of the jar as it's distributed.
> eg. maven, commons-math